test/Kestrel.FunctionalTests/ResponseTests.cs

```diff
@@ -1658,21 +1657,7 @@ public Task ResponseStatusCodeSetBeforeHttpContextDisposedRequestMalformedReadIg
                         "",
                         "");
 
-                    // Let the app finish
```

Comment:
@halter73 git thinks these should be removed. Possibly the source of failures in RequestDrainingFor100ContinueDoesNotBlockResponse in PR checks. Should we leave this out of the merge?